src/tests/rules/boolean-keywords.coffee

```diff
@@ -0,0 +1,138 @@
+###*
+# @fileoverview This rule should require or disallow usage of specific boolean keywords.
+# @author Julian Rosse
+###
+'use strict'
+
+#------------------------------------------------------------------------------
+# Requirements
+#------------------------------------------------------------------------------
+
+rule = require '../../rules/boolean-keywords'
+{RuleTester} = require 'eslint'
+path = require 'path'
+
+error = (unexpected, replacement) ->
+  if replacement?
+    messageId: 'unexpected-fixable'
+    data: {
+      unexpected
+      replacement
+    }
+  else
+    messageId: 'unexpected'
+    data: {
+      unexpected
+    }
+
+#------------------------------------------------------------------------------
+# Tests
+#------------------------------------------------------------------------------
+
+ruleTester = new RuleTester parser: path.join __dirname, '../../..'
+
+ruleTester.run 'boolean-keywords', rule,
```

Comment:
ESLint rule tests basically consist of a set of code snippets that shouldn't trigger the rule and a set of code snippets that should trigger the rule